src/commands/repos/AbstractReposCommand.ts

```diff
@@ -27,6 +29,11 @@ export abstract class AbstractReposCommand implements ICommand {
             Global.isVerbose() && console.log('running in force mode');
         }
 
+        this.sequential = !!params[AbstractReposCommand.PARAMETER_SEQUENTIAL];
+        if (this.sequential) {
+            Global.isVerbose() && console.log('running in sequential mode');
```

Comment:
This option is currently only supported by `repos -u`, and not yet documented in the help text.
How should we proceed here? Should we only document it for that command, or strive to support it in others as well?
I really like the option to run multiple operations sequentially, because it makes the logs tremendously more readable.

Reply:
I guess the motiviation of asynchronous workloads was to have better performance. Is there a significant impact on performance?
I am hardly ever using this functionality so I cant tell if it`s worth the effort making this feature available for all commands. Also I am still hoping that dependency management might be done by gradle in the near future. 
I guess it would be a good idea to officially document this feature in cli.ts.

Reply:
Doing these operations in parallel is much faster, so it should be the default. As said, understanding the logs is much harder when multiple operations are logging in parallel, so I'd like to have a sequential run option in cases something goes wrong.
I will document that the sequential option is available for the repos -u command only.

I have rarely used this command (repos -u) mainly due to its issues, like this one or that it shuffles the node_modules folder around when it is not necessary.
The need to prepare parent repos is very common when you work with another product repo, so this tool is important. Even when master is ready to depend on pre-compiled artifacts, older releases still need this. And as we have seen from some customers, they may lag a year behind and still need bugfixes for their old release.

Reply:
Regarding shuffling node_modules do we need to fix this https://github.com/collaborationFactory/cplace-npm-tools/issues/43 ?

Reply:
Or should we just remove node_modules handling from cplace-cli in general and advise the developper to manually to an `npm install` if needed?

Reply:
I will try to understand what is currently happening w.r.t. the node_modules (from the sources and the observations in #43) and what a suitable improvement would be. But not today.

I saw you started working on it, in branch `bugfix/improve-handling-of-node-modules`. :-)

Reply:
It would also be possible to do `npm install` anytime on releases >4.57. This takes about 2 additional seconds in case it was not necessary. It might also be an option to do `npm install` in cplace-asc anytime it is started on release >4.57.